eos/utils/standard-model.cc

```diff
@@ -695,6 +696,134 @@ namespace implementation
+        _mu_c__deltabs2(p["QCD::mu_c"], u),
+        _sw2__deltabs2(p["GSW::sin^2(theta)"], u),
+        _m_t_pole__deltabs2(p["mass::t(pole)"], u),
+        _m_W__deltabs2(p["mass::W"], u),
+        _m_Z__deltabs2(p["mass::Z"], u),
+        _mu_0__deltabs2(p["sbsb::mu_0"], u)
+    {
+    }
+
+    WilsonCoefficients<wc::SBSB>
+    SMComponent<components::DeltaB2>::wilson_coefficients_sbsb(const double & mu) const
+    {
+        if (mu >= _mu_t__deltabs2)
+            throw InternalError("SMComponent<components::DeltaB1>::wilson_coefficients_sbsb: Evolution to mu >= mu_t is illdefined!");
+
+        if (mu <= _mu_c__deltabs2)
+            throw InternalError("SMComponent<components::DeltaB1>::wilson_coefficients_sbsb: Evolution to mu <= mu_c is not implemented!");
+
+        // only evolve the wilson coefficients for 5 active flavors
+        static const double nf    = 5.0;
+        static const auto & beta4 = QCD::beta_function_nf_4;
+        static const auto & beta5 = QCD::beta_function_nf_5;
+        static const auto & beta6 = QCD::beta_function_nf_6;
+
+        // calculate all alpha_s values
+        const double alpha_s_mu_0 = QCD::alpha_s(_mu_0__deltabs2, _alpha_s_Z__deltabs2, _m_Z__deltabs2, beta5);
+
+        double alpha_s = 0.0;
+        if (mu < _mu_b__deltabs2)
+        {
+            alpha_s = QCD::alpha_s(_mu_b__deltabs2, _alpha_s_Z__deltabs2, _m_Z__deltabs2, beta5);
+            alpha_s = QCD::alpha_s(mu, alpha_s, _mu_b__deltabs2, beta4);
+        }
+        else
+        {
+            alpha_s = QCD::alpha_s(mu, _alpha_s_Z__deltabs2, _m_Z__deltabs2, beta5);
+        }
+
+        double alpha_s_m_t_pole = 0.0;
+        if (_mu_t__deltabs2 <= _m_t_pole__deltabs2)
+        {
+            alpha_s_m_t_pole = QCD::alpha_s(_mu_t__deltabs2, _alpha_s_Z__deltabs2, _m_Z__deltabs2, beta5);
+            alpha_s_m_t_pole = QCD::alpha_s(_m_t_pole__deltabs2, alpha_s_m_t_pole, _mu_t__deltabs2, beta6);
+        }
+        else
+        {
+            Log::instance()->message("sm_component<deltabs2>.wc", ll_error)
+                << "mu_t > m_t_pole!";
+
+            alpha_s_m_t_pole = QCD::alpha_s(_m_t_pole__deltabs2, _alpha_s_Z__deltabs2, _m_Z__deltabs2, beta5);
+        }
+
+        // calculate m_t at the matching scale in the MSbar scheme
+        const double m_t_msbar_m_t_pole = QCD::m_q_msbar(_m_t_pole__deltabs2, alpha_s_m_t_pole, 5.0);
+        const double m_t_mu_0 = QCD::m_q_msbar(m_t_msbar_m_t_pole, alpha_s_m_t_pole, alpha_s_mu_0, beta5, QCD::gamma_m_nf_5);
+
+        // calculate dependent inputs
+        const double log_t = std::log(_mu_0__deltabs2 / _m_W__deltabs2);
+        const double xt = power_of<2>(m_t_mu_0 / _m_W__deltabs2);
+        const double xt2 = xt  * xt,
+                     xt3 = xt2 * xt,
+                     xt4 = xt2 * xt2;
+        const double lnxt = std::log(xt), ln2xt = lnxt * lnxt;
+        // GSL and [BBL:1995A] convention for the dilogarithm
+        //  - gsl_sf_dilog(1.0 - x) = L_2(1.0 - x)
```

Comment:
Are you sure about the minus sign in front of gsl_sf_dilog(1.0 - x)?

Reply:
This was not meant as a minus sign, but element of a list of convention differences. In the end, none remained. I will remove the dash.

Reply:
Done